app/assets/stylesheets/pdf.css

```diff
@@ -11,7 +11,7 @@ body {
 
 * {
   font-family: "Fira Sans", sans-serif;
-  color: rgb(51, 51, 51);
+  color: #333333;
```

Comment:
You can roll back all of the changes you've made to the pdf as I've updated it all pretty significantly in my branch.

Reply:
:+1: